src/main/java/com/rakovets/course/java/core/practice/oop/principles/battle_ground_homework/Hero.java

```diff
@@ -0,0 +1,39 @@
+package com.rakovets.course.java.core.practice.oop.principles.battle_ground_homework;
+
+public abstract class Hero implements Mortal{
```

Comment:
"Violation: There isn't separating white spaces for curly brake
Solution: Add white spaces"